# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/moz.build
# Commit: 3f2605087550
# Full Hash: 3f26050875502a932b4f39326bf0f0de958441ac
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2022-06-29 03:44:30
# Regressor Bug: 1771493
# File Overlap Count: 1
# Description:
#   Bug 1771493 - Part 4. Split FontFaceSetDocumentImpl from FontFaceSetImpl. r=emilio
#   
#   This patch splits document specific functionality from FontFaceSetImpl
#   into a new class FontFaceSetDocumentImpl. This will make it easier to
#   fork FontFaceSetImpl for workers.
# ==============================================================================

diff -r 5e6bf07c9c27 -r 3f2605087550 layout/style/moz.build
--- a/layout/style/moz.build	Tue Jun 28 14:27:21 2022 +0000
+++ b/layout/style/moz.build	Tue Jun 28 14:27:21 2022 +0000
@@ -142,6 +142,7 @@
     "FontFace.h",
     "FontFaceImpl.h",
     "FontFaceSet.h",
+    "FontFaceSetDocumentImpl.h",
     "FontFaceSetImpl.h",
     "FontFaceSetIterator.h",
     "MediaList.h",
@@ -192,6 +193,7 @@
     "FontFace.cpp",
     "FontFaceImpl.cpp",
     "FontFaceSet.cpp",
+    "FontFaceSetDocumentImpl.cpp",
     "FontFaceSetImpl.cpp",
     "FontFaceSetIterator.cpp",
     "FontPreloader.cpp",
